app/controllers/api/base_controller.rb

```diff
@@ -90,5 +91,13 @@ def render_error(status, type, data = {})
         }.merge(data)
       }, status: status
     end
+
+    def log_usage!
+      Thread.new do
+        log_time = Time.current.utc.strftime('%Y-%m-%d %H:%M')
+        Exercism.redis_tooling_client.incr("#{Exercism.env}:request:url:#{request.original_fullpath}:#{request.method}:#{log_time}")
```

Comment:
I've only logged the path, not the URL, as I _think_ we're not really interested in the host name, are we?